brkraw/scripts/brkraw.py

```diff
@@ -683,5 +707,36 @@ def completeFieldsCreateFolders (df, filtered_dset, dset, multi_session, root_pa
     
     return filtered_dset
 
+
+def is_localizer(pvobj, scan_id, reco_id):
+    visu_pars = pvobj.get_visu_pars(scan_id, reco_id)
+    ac_proc = visu_pars.parameters['VisuAcquisitionProtocol']
+    if re.search('tripilot', ac_proc, re.IGNORECASE) or re.search('localizer', ac_proc, re.IGNORECASE):
+        return True
+    else:
+        return False
+
+
+def override_header(pvobj, subjtype, position):
+    """override subject position and subject type"""
+    import warnings
+    if position is not None:
+        try:
+            pvobj.override_position(position)
+        except:
```

Comment:
I think I would instead error out with incorrect.

Reply:
I put this feature for some service requests that do not provide a `subject` file.  (only provide the folder with a separate subject datasheet.
But agree that this needs to be more restricted and return error is the proper way. I will put this item for the next update.